src/lib/proxy/proxy-service.ts

```diff
@@ -16,9 +16,9 @@ import {
 import { validateOrigins } from '../util';
 import { BadDataError, InvalidTokenError } from '../error';
 import { PROXY_REPOSITORY_CREATED } from '../metric-events';
-import { ProxyRepository } from '../proxy';
-import { FrontendApiRepository } from '../proxy/frontend-api-repository';
-import { GlobalFrontendApiCache } from '../proxy/global-frontend-api-cache';
+import { ProxyRepository } from './index';
```

Comment:
Would be great if we could call this FrontendApiService and "frontend-api-service.ts"